test/powershell/Modules/Microsoft.PowerShell.Diagnostics/New-WinEvent.Tests.ps1

```diff
@@ -0,0 +1,33 @@
+Describe 'New-WinEvent' -Tags "CI" {
+
+    Context "New-WinEvent tests" {
+        
+        It 'Simple New-WinEvent' -Skip:(-not $IsWindows) {
```

Comment:
The recommendation according to the testing guidelines is to use PSDefaultParameterValues. Please see: https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md